src/scss/base/_base.scss

```diff
@@ -9,24 +9,30 @@
 
 h1 {
 	color: $grey-dark;
-	font-size: rem(48);
+	font-size: rem(35);
 	font-weight: $font-weight-semibold;
 }
 
 h2 {
 	color: $grey-dark;
-	font-size: rem(48);
+	font-size: rem(24);
 	font-weight: $font-weight-medium;
 }
 
 h3,
 .h3 {
-	font-size: rem(24);
+	font-size: rem(18);
 	font-weight: $font-weight-bold;
 }
 
-p {
+h4 {
 	font-size: rem(16);
+	font-weight: $font-weight-bold;
+}
+
+p {
```

Comment:
Instead of setting default body font size to `<p>` element, it might be better to set it to `.content-wrapper` selector, which will save many extra css for applying this default to `<ul>`, side menu, UIO table of content etc.

One problem caused by this narrow application on `<p>` elements is: on the home page, part of the main content in the bullet points is still using the browser default 16px size rather than what's on the design:

<img width="793" alt="Screen Shot 2020-07-23 at 10 43 13 AM" src="https://user-images.githubusercontent.com/572877/88300577-a2378480-ccd1-11ea-9744-5f9c96fde3c5.png">